[Issue 504] add "detail=min" parameter (#634)
astropy/pyvo: CHANGES.rst

- Fix string literal generation for SQL query when using numpy >=2.0. [#624]

> - Try minimal VOSI tables downloads by default for TAP metadata [#634]

Reviewer: Rephrasing changelog as this is not default behaviour that the user can change but a change in behaviour.

```suggestion
- Switch to do minimal VOSI tables downloads for TAP metadata. [#634]
```
